src/components/Amalgamation/AmalgamationStatement.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <v-card
+    id="amalgamation-statement"
+    flat
+  >
+    <v-row>
+      <v-col
+        cols="12"
+        sm="3"
+        class="pr-4"
+      >
+        <label>
+          Amalgamation Statement
+        </label>
+      </v-col>
+
+      <v-col
+        cols="12"
+        sm="9"
+      >
+        <v-radio-group
+          v-model="courtApproval"
+          class="mt-0 pt-0"
+          @change="setCourtApproval()"
+        >
+          <v-radio
+            class="radio-button"
+            label="With Court Approval"
+            :value="true"
+          />
+          <div
```

Comment:
When possible, you should use the best tag/element for its content -- in this case, it would be better to use the paragraph element instead of the generic div element.

See also https://www.w3schools.com/html/html5_semantic_elements.asp

Reply:
Changed to "p". Looks good. Resolving.